packages/core/entity/FileDetail/index.ts

```diff
@@ -135,6 +135,11 @@ export default class FileDetail {
     }
 
     public get localPath(): string | null {
+        // REMOVE THIS (BACKWARDS COMPAT)
+        if (this.path.startsWith("/allen")) {
```

Comment:
`this.path.startsWith("/allen")` means that it is a record from the old ETL?  

A function that describes that would be nice for readability:
`
function isOldEtlPath(path){
  retrun this.path.startsWith("/allen");
}
`